sdk/digitaltwins/azure-digitaltwins-core/CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # Release History
 
-## 1.0.0-beta.4 (Unreleased)
+## 1.0.0
 
+- Regenerate protocol layer from service API version 2020-10-31
```

Comment:
Could you section the changelog into these categories, https://azure.github.io/azure-sdk/policies_releasenotes.html#whats-a-developer-impacting-change

Reply:
Sure thing!